Assets/Editor/UnitTests/Models/ScheduledEventTest.cs

```diff
@@ -17,6 +17,7 @@ public class ScheduledEventTest
 {
     private Action<ScheduledEvent> callback;
     private bool didRun = false;
+    private int runCount = 0;
```

Comment:
Well I'm no expert either. :D It kind of makes sense to test also that it's being run the correct number of times. Using a bool could potentially miss bugs where the event is getting run too many times.